docs/INSTALL.md

```diff
@@ -100,67 +93,74 @@ Here is the list of DC/OS versions supported on dcos-terraform natively:
 
 **Note**: Master DC/OS version is not meant for production use. It is only for CI/CD testing.
```

Comment:
<p class="message--note"><strong>NOTE: </strong>Master DC/OS version is not meant for production use. It is only for CI/CD testing.</p>

Reply:
I've used HTML code to represent text as a NOTE. Please copy-paste the above.

Reply:
Done